internal/fleetfile/fleetfile.go

```
@@ -250,6 +266,69 @@ func gitCommand(ctx context.Context, root string, args ...string) *exec.Cmd {
 	return cmd
 }
 
+func Save(root string, doc *Document) error {
+	path := filepath.Join(root, FileName)
+	f, err := os.Create(path)
+	if err != nil {
+		return err
+	}
+	defer f.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `f.Close` is not checked

(errcheck)